### DISCUSSION TITLE
Writing a DDS image with BC5_UNORM format

### DISCUSSION THREAD

**Discussion by thegreatgramcracker** [dlemstra/Magick.NET#1425] [Help] [answered]
Hello, I would like to know if there is a way to write a DDS file with BC5 compression. From what I could work out, only `MagickImage.Settings.SetDefine()` was able to have any effect on the output of the file, but the only options available using that function seems to be DXT1 and DXT5, resulting in BC1 and BC3 compression respectively.

 I'm wondering if I might be missing something though because there was an [update](https://github.com/dlemstra/Magick.NET/releases/tag/13.0.0) to this library somewhat recently mentioning that BC5_UNORM was now supported. Perhaps that is read-only support? Any help or guidance is greatly appreciated.

**Comment by dlemstra** [comment, marked as answer]
There is only support for reading this format. The contribution that we got did not add support for writing in that compression.

**Reply by thegreatgramcracker** [reply]
Thank you for the clarification. Follow up question, if I wanted to make a feature request for this would it be best to do so on the original ImageMagick repository instead of this one? Since my understanding is that updates to this library typically follow updates to that one.

**Reply by dlemstra** [reply]
You should make that feature request in the ImageMagick repository. But to be honest if we don't get a pull request from someone else outside the core team of ImageMagick it will probably not be added.
